Content.Server/Power/EntitySystems/BatterySystem.cs

```diff
@@ -232,5 +235,32 @@ public bool IsFull(EntityUid uid, BatteryComponent? battery = null)
 
             return battery.CurrentCharge / battery.MaxCharge >= 0.99f;
         }
+
```

Comment:
```suggestion

        // Make sure you put this around the top of the file. I can't put this there because you didn't touch it in your PR. It goes underneath the [Dependency] block
        private const string CellContainer = "cell_slot";
```